examples/atari/atari_wrapper.py

```diff
@@ -340,7 +340,7 @@ def make_atari_env(
 
     :return: a tuple of (single env, training envs, test envs).
     """
-    env_factory = AtariEnvFactory(task, seed, frame_stack, scale=bool(scale))
+    env_factory = AtariEnvFactory(task, seed, seed + training_num, frame_stack, scale=bool(scale))
```

Comment:
```suggestion
    env_factory = AtariEnvFactory(task, seed, seed + training_num+1, frame_stack, scale=bool(scale))
```

Reply:
Add the  + 1 for consistency with test_seed property in config.py

Reply:
Actually the + 1 is not necessary. I also removed it from config.py